### ISSUE TITLE
This DIV has an aria-controls attribute value of ':r1:', which includes one or more invalid or duplicate ids

### ISSUE THREAD

**Issue by LauRoxx** [Availity/element#392]
## Issue

This DIV has an aria-controls attribute value of ':r1:', which includes one or more invalid or duplicate ids

## Details

 - Severity: 5
 - Noticeability: 4
 - Tractibility: 3

### Best Practice
[Ensure ARIA roles, states, and properties are valid](https://amp.levelaccess.net/public/standards/view_best_practice.php?violation_id=1626)

### Applicable Accessibility Standards

- Section 508 and 255 (Revised 2017)
- WCAG 2.0 Level A
- WCAG 2.0 Level A & AA Baseline
- WCAG 2.1 Level A

### Offending HTML

`<div class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-16zfmvz" id=":r0:" aria-describedby=":r0:-helper-text" aria-labelledby=":r0:-label :r0:" aria-haspopup="listbox" aria-expanded="false" aria-controls=":r1:" role="combobox" tabindex="0"><span class="notranslate">​</span></div>`

## Components/Stories Affected

- Textfield (select)
- Table Pagination
- Form Layout Stories

**Comment by LauRoxx** [comment]
This is the sibling issue to https://github.com/Availity/element/issues/213, where the options listbox is not rendered until the combobox (select) is opened, however in this case the `aria-controls` is incorrectly added to the combobox when it is not open. This is not something we can fix as a user of the library without rewriting most/all of the internals for this component

Additionally Axe, Wave, and Lighthouse do not mark this as an error while the combobox is not expanded. This makes sense when looking back to the same WAI-ARIA discussion around `aria-controls` as https://github.com/Availity/element/issues/213 we see the final note:
> The controlled element is not in the DOM but there is an aria-controls attribute on the controlling element. This is an author error that should be fixed.
> - Addendum: even if an author does not fix the error, browsers will not expose the controller-for/controlled-by relationship in the accessibility tree, because, well, there is nothing to expose. In constructing the accessibility tree, the browser will attempt to follow the (bogus) aria-controls reference, but will find nothing and will give up. The result is that there is no controller-for pointer in the accessible object. The end result is as if no aria-controls was present in the DOM. In particular, there is no pointer that could mislead a screen reader.

Essentially, while referencing a non-existent element is invalid HTML it does not affect AT in this case. 

However, the Mui team acknowledged this 3 weeks ago, and it looks like this will be fixed as they convert the select component to utilize the @mui/base library - see https://www.github.com/mui/material-ui/issues/43876. Keep in mind the fix will likely just swap out this Level Access violation for the "confirmed violation" in https://github.com/Availity/element/issues/213 - see issue for more info.
